setup-database.sh

```diff
@@ -79,7 +56,7 @@ for db in $(echo ${POSTGRES_DBNAME} | tr ',' ' '); do
         RESULT=`su - postgres -c "psql -t -c \"SELECT count(1) from pg_database where datname='${db}';\""`
         if [[  ${RESULT} -eq 0 ]]; then
             echo "Create db ${db}"
-            su - postgres -c "createdb -O ${POSTGRES_USER}  ${db}"
+            su - postgres -c "createdb -O ${POSTGRES_USER} -E ${DEFAULT_ENCODING} --lc-collate=${DEFAULT_COLLATION} --lc-ctype=${DEFAULT_CTYPE} -T template0 ${db}"
```

Comment:
If we initialize the DataDIR with locales we should not require the creation to use locales as well. This will be duplication ?

Reply:
I checked it with newly created datadir, previously the collation of the initdb parameter doesn’t change the collation of the database created. But it works if I explicitly set it like this.
I will add new unittests to demonstrate this.